sprint/index.md

```diff
@@ -1,10 +1,73 @@
 # Sprint info
 
-% TODO: arreglar toctree y texto
-
 :::{toctree}
-:maxdept: 1
+:maxdepth: 1
+:hidden:
 
+self
 sprint-guide
 sprint-cheatsheet
+faq
+resources
+:::
+
+::::{div} sd-fs-5
+:::{tip}
+* **Wondering if the sprint is for you?** We have got just {ref}`the page <decision_tree>`
+* **Ready to contribute?** Go to {ref}`docstring_tutorial` page
+:::
+::::
+
+(sprint)=
+# Open Source Working Sessions
+
+Data Umbrella is organizing a series of open source working sessions to contribute to the PyMC project. For more information, see {ref}`Resources <resources>` section.
+
+## Sprint - Goals
+
+In this sprint we’re focusing on bringing in new contributors, we sincerely hope this will be a welcoming, inclusive experience for you into the world of open source software. We hope to help you to the best of our abilities and learn from you as well!
+The sprint will consist of small resolvable issues and contributing to the PyMC documentation, so if you aren’t experienced in software development no worries at all, you can contribute to documentation with knowledge of markdown or statistics.
+Here’s the fun part, we’ll be giving away stickers, mugs and the Bayesian Computation book by PyMC Developers Ravin, Junpeng and Osvaldo!!! Find the schedule and all other details {ref}`here <current_sprint>`.
```

Comment:
This needs to be updated. I am not sure we'll be giving away another example of the book. As this is one of the static pages, I'd probably change it to "giveaways" or something like this so it is vague and then describe the specific things they will get in the schedule page as a 2nd section or something like this